tests/test_dbapi.py

```diff
@@ -122,11 +122,6 @@ def test_RollbackAfterNoChanges(self):
     def test_Cursor(self):
         cu = self.cx.cursor()
 
-    def test_FailedOpen(self):
```

Comment:
why remove?

Reply:
Forgive me. I actually just moved `test_FailedOpen` to `VariousConnectionTests` because of Connection object management.